src/portal/components/CreateIP.vue

```diff
@@ -194,16 +194,23 @@ export default class CreateIP extends Vue {
       check_ip = false;
     }
     if (this.toPublicIP.substring(0, this.toPublicIP.lastIndexOf('.')) != this.publicIP.substring(0, this.publicIP.lastIndexOf('.'))) {
+
       this.toIpErrorMessage = "IPs are not the same";
       check_same_IPs = false;
     }
     if (this.toPublicIP.split('/')[1] !== this.publicIP.split('/')[1]) {
       this.toIpErrorMessage = "Subnet is different";
       check_same_subnet = false;
     }
+
     if (parseInt(this.toPublicIP.split('/')[0].split('.')[3]) <= parseInt(this.publicIP.split('/')[0].split('.')[3])) {
+      if (parseInt(this.toPublicIP.split('/')[0].split('.')[2]) <= parseInt(this.publicIP.split('/')[0].split('.')[2])) {
```

Comment:
The ips must have the same first 3 parts. We can only change the last one.
I think the problem is with the priority of the conditions. We can move the condition of the same IDs after this condition. What do you think?